docs/pages/management/guides/awsoidc-integration.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: AWS OIDC Integration
+description: How to connect your AWS account with Teleport and provide access to AWS resources.
+---
+
+This guide explains how to set up the Teleport AWS OIDC Integration.
```

Comment:
We should mention somewhere in the beginning that users can also configure the integration using Access Management UI and this guide is for users who want to do it manually or automate via terraform.